### DISCUSSION TITLE
KeePassXC and ungoogled-chromium

### DISCUSSION THREAD

**Discussion by QuantumQuill** [Eloston/ungoogled-chromium#1632] [Q&A]
I try to get keepassxc up and running with this browser. There are a lot of people mentioning this software in the discussions but they dont seem to have any issues.
How do you use KeePass and ungoogled chromium?
Here are my issues so far: https://github.com/keepassxreboot/keepassxc-browser/issues/1391

**Comment by networkException** [comment]
I use the KeepassXC Browser extension without any issue, I just installed it form the Chrome Webstore and enabled Chromium as a Browser in the database

**Comment by QuantumQuill** [comment]
yeah well... So your workflow is:

1. Go to the Extension store [here](https://chrome.google.com/webstore/detail/keepassxc-browser/oboonakemofpalcgghocfoadofidjkkk)
2. Login into Google Website
3. Install the Extension

But I don't have a google account. and isnt all this about NOT using google?

**Reply by networkException** [reply]
You don't need a Google Account to use the Extension Store, I'm not saying its a particularly good to download from there but whatever you have done does not seem to work and in my experience this works

**Reply by QuantumQuill** [reply]
I dont see there any possibility to install without login.
![picture](https://user-images.githubusercontent.com/29311214/132094923-6c72530e-984e-41b8-9f3b-f07d8cfcdfa2.png)

**Reply by QuantumQuill** [reply]
I am on Chromium Version 88.0.4324.150 (x86_64) MacOS

**Reply by networkException** [reply]
Ah right, you will need the [chromium-web-store extension](https://github.com/NeverDecaf/chromium-web-store) (or download the crx file manually from there, [see the faq](https://ungoogled-software.github.io/ungoogled-chromium-wiki/faq#can-i-install-extensions-or-themes-from-the-chrome-webstore))
